k8s/helm/values.yaml

```diff
@@ -13,6 +13,16 @@ googleTagManagerId: ""
 googleTagManagerAuth: ""
 googleTagManagerPreview: ""
 
+webServer:
+  type: "gunicorn"
+  workerClass: "gevent"
+  workers: "7"
+  threads: "10"
+  asyncCores: "10"
```

Comment:
If this only applies to uwsgi, this should be named something like `uwsgiAsyncCores`.

Reply:
Sorry, I should have been clearer. This should be changed everywhere e.g. the env var should be `WEB_SERVER_UWSGI_ASYNC_CORES` (or `UWSGI_ASYNC_CORES`).

Reply:
I did wonder 😄 
I went with `UWSGI_ASYNC_CORES` because we have `GUNICORN_CMD_ARGS`.